app/models/katello/glue/candlepin/pool.rb

```diff
@@ -97,9 +96,6 @@ def import_data(index_hosts = false)
         pool_json = self.backend_data
 
         self.organization ||= Organization.find_by(:label => pool_json['owner']['key'])
-        product_attributes = pool_json["productAttributes"] + pool_json["attributes"]
-
-        product_attributes.map { |attr| pool_attributes[attr["name"].underscore.to_sym] = attr["value"] }
```

Comment:
This didn't appear to be used anywhere

Reply:
I think whatever was in there would have been exposed via the `pool_facts` lazy accessor